SQL/Cleanup_patches/README.md

```diff
@@ -0,0 +1,3 @@
+This directory contains SQL patches written as a post-release clean-up. These patches can contain `ALTER TABLE`, `DELETE` and/or `DROP` statements.
```

Comment:
Can we call it CleanupPatches instead of `Cleanup_patches`? I don't like the mixed case and underscore.

Reply:
s/as a/for/

Reply:
s/clean-up/cleanup/

Reply:
i was just being consistent with Release_patches.

I dont mind the change if you approve having `Release_patces` and `CleanupPatches` in the same directory